WordPress/Classes/ViewRelated/Reader/WPImageViewController.h

```diff
@@ -4,13 +4,17 @@
 
 @interface WPImageViewController : UIViewController
 
+@property (nonatomic, weak) NSNumber *index;
```

Comment:
I'd personally rather we don't add a property here just so the gallery VC can keep track of them. Perhaps we can keep an array of VCs in the gallery VC?

Reply:
I thought about going this route originally but my main reasoning around not going this was route was that I didn't want to keep tons of VCs in memory if I didn't have to.  That being said, I think there is a probably a way to not use an index in the WPImageVC and not have to store them all in an array, I'll explore that today.